app/views/layouts/_nav.html.erb

```diff
@@ -0,0 +1,11 @@
+<nav class="navbar navbar-light bg-light">
+  <a class="navbar-brand" href="/">
+    <%= image_tag("logo.png", :alt => "BookLand", :width => "100") %>
+    <h1 class="d-inline">BookLand!</h1>
+  </a>
+  <% if user_signed_in? %>
+    <%= link_to('Logout', destroy_user_session_path, method: :delete, :class => "btn btn-success") %>
```

Comment:
For consistency, I'd say you'll want to use quotation marks consistently. As with other style issues, different developer teams will have different rules on which quotation marks to use and where. As mentioned before, this is another such issue you'd let Rubocop fix so it can be done automatically.

At IEX we favour using double quotes over single quotes.

On a similar note about consistency, we should try and use the same hash syntax where possible. (EG: you have `method: :delete` and `:class => "btn btn-success"` here.

With that in mind you would change this line to:

```ruby
<%= link_to("Logout", destroy_user_session_path, method: :delete, class: "btn btn-success") %>
```

Consistency and so forth will seem really trivial and pedantic to worry about at first, but there's a good reason we pay it mind.

It causes a lot of friction in teams if they have differing styles, and as adult humans we are likely to behave like children on occasion, leading to silly arguments and the like.

If you setup these good habits early, and are flexible to change your style when you join a new team you'll never have these issues!

It also makes coding slightly easier because you always know how to layout your code and which operators to choose.

Reply:
Ah yes, I try to look out for these things in website. Noted - I'll aim for double quotes here in BookLand!